main.tf

```
@@ -0,0 +1,59 @@
+terraform {
+  required_providers {
+    clickhouse = {
+      source = "ClickHouse/clickhouse"
+      version = "0.0.2"
```

Let's either allow to use newer patch version
```suggestion
      version = "~> 0.0.2"
```
or minimum accepted version
```suggestion
      version = ">= 0.0.2"
```